src/swap/SwapScreen.tsx

```diff
@@ -230,7 +240,12 @@ export function SwapScreen({ route }: Props) {
           preparedTransactionsReviewBottomSheetRef.current?.snapToIndex(0)
           break
         case 'possible':
-          // TODO: show review screen with the possible transactions
+          // TODO: we want to remove the need to use redux, but for now keeping it
```

Comment:
when's it going to get removed?

Reply:
Should be toward the end of the refactor for [multichain support](https://linear.app/valora/project/multichain-swaps-e7ec5b514232).